Sources/ResilientDecoding/ResilientDictionary.swift

```diff
@@ -64,12 +64,12 @@ extension Decoder {
 // MARK: - Private
 
 /**
- We can't use `KeyedDecodingContainer` to decode a dictionary because it will use `keyDecodingStrategy` to map the keys, which dictionary values do not. Instead we pull out the element decoders using this wrapper type.
+ We can't use `KeyedDecodingContainer` to decode a dictionary because it will use `keyDecodingStrategy` to map the keys, which dictionary values do not.
  */
 private struct DecodingResultContainer<Success: Decodable>: Decodable {
   let result: Result<Success, Error>
   init(from decoder: Decoder) throws {
-    result =  Result {
+    result = Result {
        do {
```

Comment:
This whole section is indented one space too much FYI

Reply:
Good eye.